doc/source/whatsnew/v0.24.0.rst

```diff
@@ -106,6 +106,26 @@ Reduction and groupby operations such as 'sum' work.
 
    The Integer NA support currently uses the captilized dtype version, e.g. ``Int8`` as compared to the traditional ``int8``. This may be changed at a future date.
 
+.. _whatsnew_0240.enhancements.array:
+
+A new top-level method :func:`array` has been added for creating arrays (:issue:`22860`).
+This can be used to create NumPy arrays, or any :ref:`extension array <extending.extension-types>`, including
```

Comment:
Maybe a nitpick, but I would switch the order, to highlight its use to create extension arrays as its main purpose (I don't think people should start using it for just creating numpy arrays)